### PR TITLE
Data flow tests

### PR DESCRIPTION
**Pull request by fazamani**
this addresses #317

### CODE REVIEW

File: experiments/interpretation/dutchf3_patch/local/test.py [hunk 1]
```diff
@@ -267,28 +267,42 @@ def _evaluate_split(
...
+        output_dir = generate_path(f"{config.OUTPUT_DIR}/test/{split}", config.MODEL.NAME, current_datetime(),)
 
     running_metrics_split = runningScore(n_classes)
+   
 
     # evaluation mode:
     with torch.no_grad():  # operations inside don't track history
         model.eval()
-        total_iteration = 0
+        # total_iteration = 0
```
Comment (maxkazmsft) [inline experiments/interpretation/dutchf3_patch/local/test.py:302]: I think the comment was that this wasn't needed - if it's not needed, should we just remove it?

File: experiments/interpretation/dutchf3_patch/local/test.py [hunk 2]
```diff
@@ -267,28 +267,42 @@ def _evaluate_split(
...
+
     test_loader = data.DataLoader(test_set, batch_size=1, num_workers=config.WORKERS, shuffle=False)
 
     if debug:
+        data_flow[split]['test_loader_length'] = len(test_loader)
         logger.info("Running in Debug/Test mode")
-        test_loader = take(2, test_loader)
+        take_n = 2
+        test_loader = take(take_n, test_loader)
+        data_flow[split]['take_n_sections'] = take_n
```
Comment (maxkazmsft) [inline experiments/interpretation/dutchf3_patch/local/test.py:285]: so we're testing that there are 2 sections available for scoring, is this correct?

File: experiments/interpretation/dutchf3_patch/local/test.py (hunk 1 above)
Comment (fazamani) [reply experiments/interpretation/dutchf3_patch/local/test.py:302]: yes, I am pushing a commit that will remove this :)

File: experiments/interpretation/dutchf3_patch/local/test.py (hunk 2 above)
Comment (fazamani) [reply experiments/interpretation/dutchf3_patch/local/test.py:285]: I use this to test the length of predictions, labels, and images coming from model evaluation!
